docs/get-started/_index.md

```diff
@@ -112,14 +99,40 @@ Viam uses WebRTC and gRPC for secure peer-to-peer communications across network
 If `viam-server` can connect to the internet, you can control and monitor your machines securely **from anywhere in the world**.
 In scenarios where machines are connected to a local area network (LAN) with intermittent or no cloud connectivity, you can control and monitor machines from the local network or with code on the machines themselves.
 
+### Support
+
+The `viam-server` executable runs on **any** computer that runs on 64-bit Linux OS, macOS Computer, or Windows Subsystem for Linux.
+Here are some of the boards Viam supports:
+
+{{< board-carousel >}}
+<br>
+
+Viam also supports microcontrollers with the [micro-RDK](/installation/#install-micro-rdk).
+
+## Viam Cloud
```

Comment:
```suggestion
## Viam Cloud App
```

Reply:
or something like that

Reply:
No, this refers to the Cloud part of the platform